scripts/add-cluster.sh

```diff
@@ -5,32 +5,23 @@ set -e
 user_help () {
     echo "Creates ToolchainCluster"
     echo "options:"
-    echo "-t, --type            joining cluster type (host or member)"
-    echo "-tn, --type-name      the type name of the joining cluster (host, member or e2e)"
```

Comment:
Are any changes required in e2e for this?

Reply:
I mean specifically for the `-tn`. I don't see any changes related to `-tn` in [the e2e PR](https://github.com/codeready-toolchain/toolchain-e2e/pull/912) but maybe it just wasn't used in e2e?

Reply:
Just did a quick search, I guess it wasn't used there

Reply:
good question, it was used there, but the usage was dropped in https://github.com/codeready-toolchain/toolchain-e2e/commit/c123ef007f39d25f2f4e11c6be011f30bbb80b66#diff-163703aeb25751c1605d305fbfdf39f3a9dfb74d2ac9d8a03f5cf0169d7d41b7L242-L246
so it's not used there anymore :-)